www/index.php

```diff
@@ -597,6 +602,11 @@
                                         <label for="password" style="width: auto;">Password</label>
                                         <input type="text" name="password" id="password" class="text" style="width: 200px;">
                                     </li>
+                                    <li>
+                                           
+                                          <div class="g-recaptcha" data-callback="makeaction" data-sitekey="6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw"></div>

```

Comment:
The site key should be loaded from settings, not hard-coded.

Reply:
thankyou for help me i change 6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw to recaptcha_key
that ok?